### DISCUSSION TITLE
How to get the latest message (Python client)?

### DISCUSSION THREAD

**Discussion by Samreay** [apache/pulsar#20213] [Q&A] [answered]
Is there a way to get the latest published message in a topic efficiently using the python client? Right now in our proof of concept we've spun up  a redis instance to cache the latest value, but it seems like we're missing something obvious, because I imagine getting the latest published message is a fairly key feature in an event-driven system!

I've raised an issue with the Python client over `startMessageInclusive` (https://github.com/apache/pulsar-client-python/issues/115), but I'm aware there are probably many ways to approach things and there might be something simple I've missed.

**Comment by tisonkun** [comment, marked as answer]
Answered at https://github.com/apache/pulsar-client-python/issues/115#issuecomment-1531672113.

Closed as duplicate.
